grants.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <link rel="stylesheet" href="styles.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css"/>
+    <script src="https://kit.fontawesome.com/28783eaa78.js" crossorigin="anonymous"></script>
+    <title>Everstem Education | Grant</title>
+    <link rel="icon" href="logo.png"/>
+  </head>
+  <body>
+    <br /><br />
+<!-- Navigation -->
+<nav class="navbar" style="text-align: center;">
+  <a href="../" class="navbar-item"><img class="homeButton" src="logo.png" height=50px padding=0px></a>
+
+  <div id="side-menu">
+    <a href="../events.html" class="navbar-item home"><b>Events</b></a>
```

Comment:
events.html and index.html are in the same folder. The relative paths are not correct for the top-level menus. Suggestion: download this pull request and test it locally.